examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/khr3hvEnv.py

```diff
@@ -0,0 +1,206 @@
+        and the state of each motor.
+        :return: Observation vector
+        :rtype: numpy array
+        """
+        motorPos = self.robot.getPosition()
+        motorPos.append(self.robot.getVelocity()[0])
+
+        self.distance_walk = max(
+            self.distance_walk, self.robot.getPosition()[0])
+        self.counterLoging += 1
+        global GLOBAL_DISTANCE
+        GLOBAL_DISTANCE = max(GLOBAL_DISTANCE, self.robot.getPosition()[0])
+        # Logging on the wandb. When using Ray we don't use the custom logging because Ray has it own
+        # logging and produce compatiblity errors of two instaces of wandb.
+        if self.counterLoging % self.save_freq == 0 and self.train and not self.useRay:
+            #wandb.init(project=self.projectName, reinit=False)
+            wandb.log({"Episode distance walked": self.distance_walk,
+                       "Current x position": self.robot.getPosition()[0],
+                       "Globan distance walekd": GLOBAL_DISTANCE
+                       })
+
+        motorPos.extend([i for i in self.motorPositionArr])
+        # Shift the last 10 observations by one, on the time window of 5
+        self.obs = shift(self.obs, -self.numObs, cval=np.NaN)
+        self.obs[-self.numObs:] = motorPos
+
+        return np.array(self.obs)
+
+    def get_reward(self, action):
+        """
+        Calculate the reward of the agent. The reward award agent when moving forward on the x-axis.
+        Also, we are using a second term so the agent moves further away from his previous position.
+        :return: Reward value
+        :rtype: float
+
+        """
+        reward = 2.5 * \
+            self.robot.getPosition()[0] + \
+            self.robot.getPosition()[0] - self.prev_pos
+
+        if self.counterLoging % self.save_freq == 0 and self.train and not self.useRay:
+            wandb.log({"reward": reward,
+                       "reward-1term-weight-pos": 2.5 * self.robot.getPosition()[0],
+                       "reward-2term-diff-possition": self.robot.getPosition()[0] - self.prev_pos
+                       })
+
+        self.prev_pos = self.robot.getPosition()[0]
+        return reward
+
+    def is_done(self):
+        """
+        This method checks the termination criteria for each episode. 
+        If the criteria are satisfied return True otherwise it returns False.
+        :return: Termination criteria
+        :rtype: Boolean
+
+        """
+        # Agent fall.
+        if self.robot.getPosition()[2] < 0.5:
+            return True
+        # Agent walked out of the box
+        if self.robot.getPosition()[0] > 15:
+            return True
+        return False
+
+    def get_default_observation(self):
+        """
+        Simple implementation returning the default observation which is a zero 
+        vector in the shape of the observation space.
+        :return: Starting observation zero vector
+        :rtype: numpy array
```

Comment:
Based on [this](https://numpy.org/doc/stable/reference/generated/numpy.zeros.html) ("returns" section)
```suggestion
        :rtype: ndarray
```